.github/workflows/first_issue_reply.yml

```diff
@@ -0,0 +1,20 @@
+name: First Issue Reply
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # runs every day at midnight
```

Comment:
Why can't this just be triggered by someone making a PR/issue?

Reply:
Reworked to run every time a comment is posted on an issue.